Sources/IO/Misc/OBJReader/index.js

```diff
@@ -195,12 +195,12 @@ function vtkOBJReader(publicAPI, model) {
     model.compression = option.compression;
 
     // Fetch metadata
-    return publicAPI.loadData();
+    return publicAPI.loadData({progressCallback: option.progressCallback});
   };
 
   // Fetch the actual data arrays
-  publicAPI.loadData = () => {
-    const promise = fetchData(model.url);
+  publicAPI.loadData = (option = {}) => {
+    const promise = fetchData(model.url, {progressCallback: option.progressCallback});
```

Comment:
Would work for now, but in the future the wrapped method may take different options or just a subset of the options from the outer method. If you prefer that way, though, I will change it.